Codeinwp/neve
# Feat single page globals

### Summary
Add global theme mods for the single page.

### Will affect visual aspect of the product
NO

### Test instructions
- Go to customizer -> Layouts -> Single page
- Check that all controls are working
- Check if the single options are overriding the global ones


Closes Codeinwp/neve-pro-addon#1569.
